wikidataStuff/WikidataStuff.py

```diff
@@ -228,37 +253,64 @@ def addLabelOrAlias(self, lang, name, item, summary=None,
         @param summary: summary to append to auto-generated edit summary
         @param caseSensitive: if the comparison is case sensitive
         """
-        summary = summary or self.edit_summary
+        data = {lang: name}
+        self.add_multiple_label_or_alias(
+            data, item, case_sensitive=caseSensitive, summary=summary)
+
+    def add_multiple_label_or_alias(self, data, item, case_sensitive=False,
```

Comment:
Hm, was the decorator problem caused by the typo in the last commit? What's the status now?